UPGRADE-7.1.md

```diff
@@ -65,6 +65,42 @@ as service.
 
 All service classes that are loaded by `ServiceLocator::get()` must now be `public="true"` in their service definitions.
 
+## Correct Cronjob Hierarchy
+
+Cronjob classes need to extend the class TdbCmsCronjobs. Currently the often used class TCMSCronJob does not have all the necessary field.
+So searching for "TCMSCronJob" in the project code should give you a complete list for this task.
+Furthermore every cronjob should only calls the parent constructor like so:
+
+Before:
+```
+class ... extends TCMSCronJob {
+
+ public function __construct()
+    {
+        parent::TCMSCronJob();
+    }
+}
+
+// or
+
+class ...Cronjob extends TdbCmsCronjobs {
+
+public function __construct()
+    {
+        parent::TdbCmsCronjobs();
+    }
+}
+```
+Should Be:
+```
+class ...Cronjob extends TdbCmsCronjobs {
+
+ public function __construct()
```

Comment:
(Empty constructors are not needed.)

So proably we should make clear that there is (other) code in the constructor.

Reply:
constructor example updated with demo parameters.